data/secscan_model/secscan_v4_model.py

```diff
@@ -450,28 +452,70 @@ def should_skip_indexing(manifest_candidate):
                         if vulnerability_report is not None:
                             found_vulnerabilities = vulnerability_report.get("vulnerabilities")
 
+                        # Current implementation creates events for all detected vulnerabilities, including vulnerabilities that are low
+                        # or unknown severity, which makes the whole thing a bit pointless. We'll filter out all vulnerabilities that aren't
+                        # high or critical.
+                        level = (
+                            self.app.config.get("NOTIFICATION_MIN_SEVERITY_ON_NEW_INDEX")
+                            if self.app.config.get("NOTIFICATION_MIN_SEVERITY_ON_NEW_INDEX")
+                            else "High"
+                        )
+                        lowest_severity = PRIORITY_LEVELS[level]
+
                         if found_vulnerabilities is not None:
                             import notifications
 
+                            logger.debug(
+                                "Attempting to create notifications for manifest %s", manifest
+                            )
+
                             keys = list(found_vulnerabilities)
                             for key in keys:
                                 vuln = found_vulnerabilities[key]
 
-                                event_data = {
-                                    "vulnerable_index_report_created": "true",
-                                    "vulnerability": {
-                                        "id": vuln["id"],
-                                        "description": vuln["description"],
-                                        "link": vuln["links"],
-                                        "priority": vuln["severity"],
-                                        "has_fix": bool(vuln["fixed_in_version"]),
-                                    },
-                                }
-
-                                notifications.spawn_notification(
-                                    manifest.repository, "vulnerability_found", event_data
+                                found_severity = PRIORITY_LEVELS.get(
+                                    vuln["normalized_severity"], PRIORITY_LEVELS["Unknown"]
                                 )
 
+                                if found_severity["score"] >= lowest_severity["score"]:
+                                    tag_names = list(
+                                        registry_model.tag_names_for_manifest(manifest, TAG_LIMIT)
+                                    )
+                                    if tag_names:
+                                        event_data = {
+                                            "tags": list(tag_names),
```

Comment:
nit: could we do something like
```
"tags": list(tag_names) if tag_names else [manifest.digest]
```
to avoid having to write out `event_data` twice?

Reply:
@Marcusk19 I actually tried that as my first thought. And then what I got is a bunch of notifications for `s`, `n`, `a`... i.e. the whole SHA digest (each letter and number) was interpreted as a tag name. This does not happen with the two different events.